Source/WebCore/inspector/InspectorNodeFinder.cpp

```diff
@@ -91,8 +91,8 @@ void InspectorNodeFinder::searchUsingDOMTreeTraversal(Node& parentNode)
         case Node::ELEMENT_NODE:
             if (matchesElement(downcast<Element>(*node)))
                 m_results.add(node);
-            if (is<HTMLFrameOwnerElement>(downcast<Element>(*node)))
-                performSearch(downcast<HTMLFrameOwnerElement>(*node).contentDocument());
+            if (RefPtr frameOwner = dynamicDowncast<HTMLFrameOwnerElement>(*node))
```

Comment:
The RefPtr here is not really necessary as we only call a trivial getter on frameOwner.